### ISSUE TITLE
Can't add arbitrary account to proxy

### ISSUE THREAD

**Issue by lucasvo** [polkadot-js/apps#3834]
I'm trying to add `12QvUtfFfGF6WmDT1dfDxe3fdHD57dDUF9UtrY1Gi6cR2G7c` as to an anonymous proxy `15vkPmkEmTEQCUnq8SztRDGeyvXdCjeX28Yso8V3v8G8bhNV` on https://polkadot.js.org/apps/#/accounts but it doesn't let me select anything in the dropdown and there is no console output.

![screen_2020-10-09-16-18-36](https://user-images.githubusercontent.com/31727/95607398-36042b80-0a4b-11eb-9fcb-c87688b823a1.png)

**Comment by jacogr** [comment]
From a UI pov it only allows adding own accounts. Opening it for any address leads to users shooting themselves in the foot so it is unlikely to change.

**Comment by lucasvo** [comment]
I don't really agree with this decision. Our use case for proxies is that our team controls one proxy where we can add remove users. Don't you think this is quite common? Forcing users to add foreign accounts through an extrinsic is equally dangerous I think.

At least this should be presented with a proper error message.

**Comment by jacogr** [comment]
I understand you don't agree with that view, but as mentioned, it is unlikely to change.

**Comment by lucasvo** [comment]
If I look at the polkadot council about half the seats are taken by "organizations", those are accounts where it's unlikely that one person has all of these accounts that should be set on the proxy are from controlled on a single computer, the whole point of a proxy is that you can *distribute* control. Why do you think it's worth it to force people to manually submit an extrinsic to for this very common use case?

**Comment by jacogr** [comment]
I'm not going to argue around this. Generally since I write bulk of the code, and use the apps across networks, I focus on -

(a) things that I personally use on a daily basis (so there is very much a slant to stuff I use daily)
(b) things where there are enough requests as an enhancement request
(c) where there are general painpoints that keep standing in the way (generally bugs, sometimes additional info to smooth use) 

There is x hours to spend on stuff and I'd rather say no. As always, happy to re-visit when either (a), (b) or (c) are met and it won't create a support problem. So, I take your points, but sadly none of it fits in something that I am willing to deal with support or maintenance wise at this point.

I always appreciate feedback, like here, this is actually the second request now, in both cases from people I'd consider "knowledgeable users" (which does add some weight), but it is not going to change short-term.

**Comment by lucasvo** [comment]
@jacogr of course, also that there are many things that probably have priority. I'm also knowledgeable enough that after reading the docs on the extrinsics submitting it manually was no issue. The main reason I filed this is because I wanted to make sure that it's tracked as an issue. 

It was also really something that wasn't intuitive at all for me so I spent a lot of time wondering why I couldn't just enter an arbitrary address and the empty dropdown made me think this was a UI bug not an intended feature. Maybe just saying in the dropdown when you enter an invalid address that it's not a known account would be helpful in general (also other places where an account must be selected).
